core/src/edu/asu/c3simulator/util/PlayerInterfaceImplementation.java

```diff
@@ -0,0 +1,89 @@
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface Provides detail about various player
+ * attributes that are needed for PlayerStatusDisplay
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerInterfaceImplementation implements Player
+{
+	private SocioeconomicStatus status;
+	private int netWorth;
+	private int capital;
+	private C3Simulation simulation;
+
+	public PlayerInterfaceImplementation(SocioeconomicStatus newStatus, int newNetWorth,
+			int newCapital, C3Simulation newSimulation)
```

Comment:
Avoid names like: "newStatus" "newBleh" "new..." etc. Instead, use the name of the data member (e.g., status, netWorth, etc) and specify the class-scope by using this.status, this.netWorth, etc.